### PR TITLE
Add Lennart and Adam as reviewers

### PR DESCRIPTION
**Pull request by kashifest**
Although, community guideline is to have 10 commits for a reviewer, this particular repository doesn't see that much commit on its own. I believe Lennart and Adam has necessary knowledge on this repo and they can be of help while reviewing codes in this repo.

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -8,12 +8,14 @@ approvers:
  - iurygregory
 
 reviewers:
- - namnx228
  - zaneb
+ - lentzi90
+ - Rozzii
```
Comment (dtantsur) [inline OWNERS:13]: nit: keep sorted (ditto below)

Comment (kashifest) [reply OWNERS:12]: Thanks done.

**Comment by kashifest** [comment]
I would really love to skip tests on such changes :( 
/test-ubuntu-integration-main

**Pull request merged**